department-of-veterans-affairs/next-build
# [DNM] VACMS-9829 facility menu component

## Description
Closes #9829. Adds support for the facility side nav to our story listing page. adds some light docs. I will clean the rest of this description up later when this isn't a draft PR.

## Testing done

## Screenshots

## QA steps
- visit http://localhost:3000/pittsburgh-health-care/stories and see the menu exists.

### Definition of Done
